cdk/lib/admin-console.ts

```diff
@@ -12,39 +12,31 @@ import {
 } from '@guardian/cdk/lib/constructs/iam';
 import type { App, CfnElement } from 'aws-cdk-lib';
 import { Duration, RemovalPolicy } from 'aws-cdk-lib';
-import { AttributeType, ProjectionType, Table } from 'aws-cdk-lib/aws-dynamodb';
+import { AttributeType, Table } from 'aws-cdk-lib/aws-dynamodb';
 import { InstanceClass, InstanceSize, InstanceType } from 'aws-cdk-lib/aws-ec2';
 
 export interface AdminConsoleProps extends GuStackProps {
   domainName: string;
 }
 
 export class AdminConsole extends GuStack {
-  // Build a dynamodb table to store tests for the given channel
-  buildTestsTable(channel: string): Table {
-    const id = `${channel[0].toUpperCase() + channel.slice(1)}TestsDynamoTable`;
+  // Build a dynamodb table to store tests for all channels
+  buildTestsTable(): Table {
+    const id = `ChannelTestsDynamoTable`;
 
     const table = new Table(this, id, {
-      tableName: `${channel}-tests-${this.stage}`,
+      tableName: `channel-tests-${this.stage}`,
       removalPolicy: RemovalPolicy.RETAIN,
       readCapacity: 4,
       writeCapacity: 4,
       partitionKey: {
-        name: 'name',
+        name: 'channel',
         type: AttributeType.STRING,
       },
-    });
-
-    // Add an index for querying by status (LIVE/DRAFT/ARCHIVED)
-    table.addGlobalSecondaryIndex({
```

Comment:
for now let's try without any indexes

Reply:
yeah makes sense to add an index alongside building the feature that will use it 👍